src/ingest_llm_as/api/ingestion.py

```
@@ -89,6 +99,43 @@ async def ingest_text(
+        raw_record = RawIngestion(
+            ingestion_id=ingestion_id,
+            source_type="text",
+            content_type=request.metadata.content_type.value,
+            raw_payload=request.model_dump(mode="json"),
+            raw_metadata={
+                "source": request.metadata.source.value,
+                "tags": request.metadata.tags,
+                "source_url": request.metadata.source_url,
+                "title": request.metadata.title,
+            },
+            captured_at=datetime.utcnow(),
```

Ruff (0.14.11)

`datetime.datetime.utcnow()` used

(DTZ003)